Targets/Spidermonkey/fuzzbuild.sh

```diff
@@ -22,6 +22,6 @@ export CXX=clang++-10
 
 mkdir fuzzbuild_OPT.OBJ
 cd fuzzbuild_OPT.OBJ
-/bin/sh ../configure.in --enable-debug --enable-optimize --disable-shared-js
+/bin/sh ../configure.in --enable-debug --enable-optimize --disable-shared-js --enable-js-fuzzilli
 
-make -j
+make -j8
```

Comment:
Is this change intentional?

Reply:
Yeah, i've noticed it forces my virtual machine to reset. Xubuntu 20.04 on Virtual Box.

Reply:
Ok that's non ideal, limiting it to 8 will probably make building on the 72 core machine I'm using awfully slow :D How about querying the number of available CPUs, e.g. via `getconf _NPROCESSORS_ONLN` (seems to work on Linux and macOS) and maybe falling back to 8 if that returns an empty string?

Reply:
I get it :)
Sounds good.